src/Pixie/QueryBuilder/QueryBuilderHandler.php

```diff
@@ -47,14 +47,15 @@ class QueryBuilderHandler
      *
      * @var array
      */
-    protected $fetchParameters = array(\PDO::FETCH_OBJ);
+    protected $fetchParameters = array(PDO::FETCH_OBJ);
 
     /**
      * @param null|\Pixie\Connection $connection
      *
-     * @throws \Pixie\Exception
```

Comment:
you mean this @TCB13 ?
this is because you are using `use Pixie\Exception`, this will be automatically inferred to be a Pixie one